README.md

```diff
@@ -33,11 +33,15 @@ directories will exist:
 Auto-generated developer documentation can be found at our corresponding
 [doclets site](https://doclets.io/bookbrainz/bookbrainz-site/master). Our
 contributing guide can be found [here](CONTRIBUTING.md).
-
-
-<br/>
 <br/>
+## Testing subdomain
 
+A separate subdomain for the whole purpose of testing can be found [here](https://test.bookbrainz.org). 
+You'll have to use a separate account for this subdomain. All changes made to this subdomain are not in sync with the original website and vise versa (each has its own database). 
```

Comment:
> You'll have to use a separate account for this subdomain.

You can sign in with the same account as the one you use on the main website.
The rest of the sentence is very clear.